grakn-engine/src/main/java/ai/grakn/engine/task/postprocessing/IndexPostProcessor.java

```diff
@@ -115,16 +115,16 @@ public void mergeDuplicateConcepts(EmbeddedGraknTx<?> tx, String conceptIndex, S
      * Checks that post processing was done successfully by doing two things:
      *  1. That there is only 1 valid conceptID left
      *  2. That the concept Index does not return null
-     * @param tx A grakn graph to run the checks against.
+     * @param graph A grakn graph to run the checks against.
```

Comment:
Whoops! Looks like we reverted something 😛

Reply:
fixed!